mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -520,6 +488,81 @@ class ALSSuite
   }
 }
 
+class ALSCleanerSuite extends SparkFunSuite {
+  test("Clean shuffles") {
```

Comment:
This doesn't match the style of the rest of the tests in this file, want me to update all of them?